lib/attr_encrypted.rb

```diff
@@ -356,6 +356,10 @@ def encrypted_attributes
 
     protected
 
+      def not_empty?(value)
+        !value.nil? && !(value.is_a?(String) && value.empty?)
+      end
```

Comment:
You should follow the DRY principle and not repeat a method that's already defined here:
https://github.com/XanderPSON/attr_encrypted/blob/979eecf1f1acacb72f59f1fbfe89d38f9510257d/lib/attr_encrypted.rb#L359-L361

Reply:
I tried to call the method already defined but it was namespaced under a different module. I tried various ways to reference it but I wasn't successful. Any suggestions on how to make that method call without duplicating it would be greatly appreciated :)